starcheck/calc_ccd_temps.py

```diff
@@ -153,10 +160,10 @@ def get_ccd_temps(oflsdir, outdir='out',
     proc['datestart'] = DateTime(tstart).date
     proc['datestop'] = DateTime(tstop).date
 
-    # Get temperature telemetry for 30 days prior to min(tstart, NOW)
+    # Get temperature telemetry for 7 days prior to min(tstart, NOW)
     tlm = get_telem_values(min(tstart, tnow),
-                           ['aacccdpt', 'pitch'],
-                           days=30)
+                           ['aacccdpt'],
```

Comment:
I cut the pitch fetching because it both would not really work on maude and because I don't recall why it was being fetched in the first place.

Reply:
Not sure either.  Pitch is obviously a requirement for computing the model, but that comes from the states here (if I understand).